src/app/shared/navigation/navigation.component.ts

```diff
@@ -83,12 +91,23 @@ export class NavigationComponent {
   ];
 
   public constructor(
+    private readonly featureStateResolver: FeatureStateResolver,
     private readonly navigationListService: NavigationListService,
     private readonly preferenceService: PreferenceService,
     private readonly navListComponentService: NavigationListComponentService,
     private readonly activatedRoute: ActivatedRoute,
     private readonly timeRangeService: TimeRangeService
   ) {
+    this.subscriptions.add(
```

Comment:
You missed `SubscriptionLifecycle` here 😅

Reply:
Done ✅